File: back/src/main/java/mitei/mitei/political/balancesheet/manage/kanrensha/dto/wktbl_std/UpdateWkTblStdCorpResultDto.java
package mitei.mitei.political.balancesheet.manage.kanrensha.dto.wktbl_std;

import java.io.Serializable;

import mitei.mitei.political.balancesheet.manage.kanrensha.dto.FrameworkMessageAndResultDto;
import mitei.mitei.political.balancesheet.manage.kanrensha.entity.WkTblMasterCorpEntity;

/**
 * 企業／団体一括登録マスタ標準ワークテーブル更新CapsuleDto
 */
public class UpdateWkTblStdCorpResultDto extends FrameworkMessageAndResultDto // NOPMD DataClass
        implements Serializable {

    /** Serialize id */
    private static final long serialVersionUID = 1L;

    /** 編集対象Entity */
    private WkTblMasterCorpEntity wkTblMasterCorpEntity = new WkTblMasterCorpEntity();

    /**
     * 編集対象Entityを取得する
     *
     * @return 編集対象Entity
     */
    public WkTblMasterCorpEntity getWkTblMasterCorpEntity() {
        return wkTblMasterCorpEntity;
    }

    /**
     * 編集対象Entityを設定する
     *
     * @param wkTblMasterCorpEntity 編集対象Entity
     */
    public void setWkTblMasterCorpEntity(final WkTblMasterCorpEntity wkTblMasterCorpEntity) {
        this.wkTblMasterCorpEntity = wkTblMasterCorpEntity;
    }

}

File: front/src/dto/wktbl_std/updateWkTblStdCorpResultDto.ts
import type WkTblMasterCorpInterface from "../../entity/wkTblMasterCorpEntity";
import WkTblMasterCorpEntity from "../../entity/wkTblMasterCorpEntity";
import FrameworkMessageAndResultDto from "../frameworkMessageAndResultDto";

export default interface UpdateWkTblStdCorpResultInterface {

}

/**
 * 企業／団体一括登録マスタ標準ワークテーブル更新ResultDto
 */
export default class UpdateWkTblStdCorpResultDto extends FrameworkMessageAndResultDto
    implements UpdateWkTblStdCorpResultInterface {

    /** 編集対象Entity */
    wkTblMasterCorpEntity: WkTblMasterCorpInterface;

    constructor() {
        super();
        this.wkTblMasterCorpEntity = new WkTblMasterCorpEntity();
    }
}
